zhmccli/_cmd_adapter.py

```diff
@@ -28,6 +31,31 @@
 from ._cmd_cpc import find_cpc
 
 
+# Mapping of --crypto-type option values to crypto-type adapter property values
+CRYPTO_TYPE_PROPERTIES = {
+    'acc': 'accelerator',
+    'cca': 'cca-coprocessor',
+    'ep11': 'ep11-coprocessor',
+}
+
+# List of --crypto-type option values
+CRYPTO_TYPE_OPTIONS = list(CRYPTO_TYPE_PROPERTIES.keys())
```

Comment:
are they sorted by default? should be.

Reply:
I would prefer to keep the order defined in the dictionary, but that does not work below Python 3.6 (unless we change to OrderedDict).
 
 Changed this to sort the list, and also for FICON_TYPE_OPTIONS.